managers/asset-operations-manager.js

```diff
@@ -72,17 +91,43 @@ class AssetOperationsManager {
       tokenId
     );
 
-    const operationId = await this.nodeApiService.publish(
-      PUBLISH_TYPES.ASSET,
-      assertionId,
-      assertion,
-      UAL,
-      options
-    );
+    let operationResult;
+    let operationId;
+    if (privateAssertionId && !utilities.isEmptyObject(privateAssertion)) {
+      operationId = await this.nodeApiService.localStore(
+        [
+          { assertionId: privateAssertionId, assertion: privateAssertion },
+          { assertionId: publicAssertionId, assertion: publicAssertion },
+        ],
+        options
+      );
+      operationResult = await this.nodeApiService.getOperationResult(
+        operationId,
+        { ...options, operation: OPERATIONS.LOCAL_STORE, frequency: 0.5 }
```

Comment:
Can we pass a frequency as option parameter for the call, Also can you move the default value in constants